test/testcommon/src/main/java/io/pravega/test/common/PasswordAuthHandlerInput.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.common;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import lombok.Data;
+import lombok.Getter;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This is a helper class for tests and may be used for generating the input file for the PasswordAuthHandler - the
+ * default AuthHandler implementation.
+ */
+public class PasswordAuthHandlerInput {
```

Comment:
This class is currently used only in integration tests, and as such, there is no need to share. It also has the issue we discussed in the other PR that `test:common` is not the place for Pravega-specific classes.

I'd rather have it in `test:integration` for the time being.

Reply:
I kept it here as I am aware of existing test classes other than integration tests that can benefit from it (`ControllerGrpcAuthFocusedTest` is a case in point). I have not updated those existing test classes right now, though. I was thinking we'll move it to the shared project later when we issue #3745. 
What do you say?

Reply:
Let's not abuse the exception. There is no need for this class to be here, and if there is a need to share it, which is not the case in this PR, let's do the work of enabling `shared:test`.

Reply:
:) Sure. As suggested, I've moved this class to the project `test:integration` under the package `io.pravega.test.integration.utils`. Hope that works.